kuma/javascript/src/payments/pages/thank-you.jsx

```diff
@@ -42,10 +42,10 @@ const ThankYouPage = ({ locale }: Props) => {
                                     )}
                                     subscriptionLink={
                                         <a
-                                            href={`/${locale}/payments/recurring/management/`}
+                                            href={`/${locale}/payments/management/`}
                                         >
                                             {gettext(
-                                                'manage monthly subscription page'
+                                                'manage monthly subscriptions page'
```

Comment:
Another instance of the "subscriptions" vs "subscription" question.

Reply:
Changed to singular